ownership-chain/primitives/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "ownership-parachain-primitives"
 description = "Primitives of Ownership parachain runtime."
-version = "0.1.0"
+version = { workspace = true }
```

Comment:
that's very unnecessary, imo. We make a lot of changes to pallets/precompiles, and it wouldn't make sense to bump the version whenever it's modified.

I changed it here because versions of these packages were everywhere. Some of them are straight up copied from somewhere. But I will revert them and only leave runtime/node pacakage versions